netci.groovy

```diff
@@ -113,6 +113,96 @@ def CreateBuildTasks = { machine, configTag, buildExtra, testExtra, runCodeAnaly
     }
 }
 
+def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSetup ->
+    [true, false].each { isPR ->
+        ['debug', 'release'].each { buildType ->
+            def config = "linux_${buildType}"
+            config = (configTag == null) ? config : "${configTag}_${config}"
+
+            // params: Project, BaseTaskName, IsPullRequest (appends '_prtest')
+            def jobName = Utilities.getFullJobName(project, config, isPR)
+
+            def infoScript = '''
```

Comment:
I would pull this out into your repo as a separate script, just to avoid too much hidden scripting in the CI system

Also, 'echo.' will print a plain newline :)